src/layouts/BaseLayout.astro

```diff
@@ -9,7 +9,8 @@ const title = "MockUPhone";
 const description =
   "MockUPhone is a free tool that helps you wrap app screenshots in different mobile devices. Support iPhone mockup, iPad mockup, Android mockup and TV mockup.";
 const social_url = "home_url";
-const social_icon = "/Images/mockuphone-logo.png";
+const social_icon =
+  "https://github.com/oursky/mockuphone.com/blob/main/public/Images/mockuphone-logo.png?raw=true";
```

Comment:
Sor, on second thought - are we sure we want to rely on below?

1. `mockuphone.com` repo being public 
2. use github as image hosting provider

Maybe a better approach would be to use `https://mockuphone.com/Images/mockuphone-logo.png`?

Reply:
Even more robust - make sth like below?

```
# .env.production
BASE_URL = `https://mockuphone.com`
# .env.staging
BASE_URL = `https://staging.mockuphone.com`
```